core/store/src/db/rocksdb.rs

```diff
@@ -382,14 +382,46 @@ fn rocksdb_read_options() -> ReadOptions {
 fn rocksdb_block_based_options(
     block_size: bytesize::ByteSize,
     cache_size: bytesize::ByteSize,
+    col: DBCol,
 ) -> BlockBasedOptions {
     let mut block_opts = BlockBasedOptions::default();
+    block_opts.set_checksum(/*no checksum */ '\0');
     block_opts.set_block_size(block_size.as_u64().try_into().unwrap());
     // We create block_cache for each of 47 columns, so the total cache size is 32 * 47 = 1504mb
-    block_opts
-        .set_block_cache(&Cache::new_lru_cache(cache_size.as_u64().try_into().unwrap()).unwrap());
-    block_opts.set_pin_l0_filter_and_index_blocks_in_cache(true);
-    block_opts.set_cache_index_and_filter_blocks(true);
+    if col == DBCol::BlockHeight
+        || col == DBCol::ChunkHashesByHeight
+        || col == DBCol::HeaderHashesByHeight
+        || col == DBCol::BlockPerHeight
+        || col == DBCol::BlockRefCount
+    {
+        // num_shards_bits = 0 will lead to LRU cache having (1 << 0) = 1 shards.
+        block_opts.set_block_cache(
+            &Cache::new_lru_cache_with_shard_bits(
+                cache_size.as_u64().try_into().unwrap(),
+                /*num_shard_bits */ 0,
```

Comment:
```suggestion
                0,
```
I think the comment above should suffice.

Reply:
I'm not quite sure. IMO it's easier to read

Reply:
At least add space after `/*` ;)